RealmConverter/Exporter/DataExporter.swift

```diff
@@ -26,7 +26,7 @@ import Realm
 */
 open class DataExporter: NSObject {
     
-    open let realm: RLMRealm
+    @objc public let realm: RLMRealm
```

Comment:
Any particular reason to reduce the access level (here and below)?

Reply:
Swift generates an error when this is `open`:
    `'let' properties are implicitly 'final'; use 'public' instead of 'open'`